code/dqe/xmarketalert.q

```diff
@@ -0,0 +1,9 @@
+\d .dqe 
+xmarketalert:{[tab]                                     // alerts user when bid has exceeded the ask in market data
+  $[0b=all `bid`ask in cols tab;
+    (0b;"columns bid and ask do not exist in provided table");
+    (data:select from tab where bid>ask;
+    $[0=count data;
```

Comment:
This second condition should probably be wrapped in square brackets to make it a series of statements rather than a list but it can be made into a single statement instead:
```suggestion
    $[0=count data:select from tab where bid>ask
```

Reply:
I was testing it out the other day and it turns out the error handling of DQE handles the error mentioned by Jonathon earlier and no further conditions were needed